wdl-grammar/src/lib.rs

```diff
@@ -35,7 +35,7 @@ mod diagnostic;
 pub mod grammar;
 pub mod lexer;
 pub mod parser;
-mod tree;
+pub mod tree;
```

Comment:
cc: @peterhuene, just making sure you're okay with this

Reply:
For my understanding, why was this change required? We're already publicly using everything in the tree module at the top-level.

Reply:
Hm, I'm not sure looking in hindsight. I suppose that was something I changed and forgot to remove this once it was unneeded.